.github/ISSUE_TEMPLATE/docs.md

```diff
@@ -0,0 +1,14 @@
+---
```

Comment:
It seems weird to have a table in here, or does GitHub do something special with it?

Reply:
GitHub automatically picks the information up when showing the available issue templates.

Reply:
Yeah - it's a format that GitHub uses to get metadata https://help.github.com/en/articles/manually-creating-a-single-issue-template-for-your-repository (see step 5)